### PR TITLE
Mark transform_snapping as experimental in class ref

### PR DESCRIPTION
**Pull request by lawnjelly**
Alternative for 3.2.4, if we keep the existing functionality that may not be guaranteed to work correctly.

### CODE REVIEW

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -1047,7 +1047,7 @@
 			[b]Note:[/b] Antialiased software skinned polys are not supported, and will be rendered without antialiasing.
 		</member>
 		<member name="rendering/2d/snapping/use_camera_snap" type="bool" setter="" getter="" default="false">
-			If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
+			[b]Experimental[/b] If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
```
Comment (akien-mga) [inline doc/classes/ProjectSettings.xml:1050]:
```suggestion
			[b]Experimental:[/b] If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
```
(Same for the other one.)

But what I would do (additionally to this) is actually put "experimental" in the setting name:
`rendering/2d/snapping/experimental_use_camera_snap` or something.

Comment (lawnjelly) [reply doc/classes/ProjectSettings.xml:1050]: Ok will do. :+1:

**Comment by lawnjelly** [comment]
Actually, waking up this morning I had a much better idea for compromise.

If we revert the snapping in core, I can add it to my smoothing addon (which is widely used anyway). That way we will have more time to refine it, it will be available to those who want it and we won't get bogus support issues. Then if we come to a good solution we can add it back in core in 3.2.5.

It will actually be easier for me to test it by the numbers via an addon anyway. That would have been a better approach for coming up with the best solution for core.

**Comment by akien-mga** [comment]
Superseded by #46657.

**Pull request closed without merging**